client/README.md

```diff
@@ -66,3 +66,23 @@ This section has moved here: https://facebook.github.io/create-react-app/docs/de
 ### `npm run build` fails to minify
 
 This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+
+
+
+
+
+Why would you use class component over function components (removing hooks from the question)?
+Only if im working on a project started by another developer and they build the components using class component.
+
```

Comment:
Class components are an alternative to using hooks where you want to instantiate state within a component.